automated install/uninstall.sh

```diff
@@ -14,7 +14,6 @@ while true; do
     read -rp "  ${QST} Are you sure you would like to remove ${COL_WHITE}Pi-hole${COL_NC}? [y/N] " yn
     case ${yn} in
```

Comment:
Need to take out the `n` option here as well.

Reply:
Why? `yn` is only the name of the variable

Reply:
maybe `response` or `answer` instead? 🤷‍♂️

Reply:
Done.